src/runtime_src/core/pcie/driver/windows/include/XoclMgmt_INTF.h

```diff
@@ -109,12 +117,6 @@ typedef struct {
     CHAR   ShellFilename[64];
     CHAR   BMCVersion[16];
     UINT32 VlanTag;
-    CHAR   MacAddr0[32];
```

Comment:
why was this removed? I am under the impression, that MS tools(azxbutil) was expecting this information. You might want to double check it.

Reply:
Yes, added MacAddress back to the struct. It was removed as part of https://github.com/Xilinx/XRT/pull/5476 actually. Now, fixed it here.